Update APR calcucation.md

bro-n-bro-2 opened the pull request:
fixed Osmosis, Evmos.
Added Crescent
All other chains was double checked. They are using standard formula.

bro-n-bro-2 committed:
Update APR calcucation.md

bro-n-bro-2 committed:
Update APR calcucation.md

bro-n-bro-0 commented:
1. Please use camelCase or underscore_case for variables in equations

2. It is possible to simplify APR equation in general if 
```
inflation_rate = annual_provisions / total_supply
```
and
```
bonded_tokens_ratio = bonded_tokens_amount / total_supply
```
the general APR equation can be 

```
APR = annual_provisions * (1 - community_tax) / bonded_tokens_amount
```

This way we can unify epoch and non-epoch networks

For non-epoch networks the `annual_provisions` is usually available by the `/cosmos/mint/v1beta1/annual_provisions` endpoint

for epoch networks the `annual_provisions` is

```
annual_provisions = epoch_provisions * 365.3
```

Also, you can add a `correction_annual_coefficient` to non-epoch networks

This is an relation between `real_blocks_per_year` and `block_per_year` in params

```
correction_annual_coefficient = real_blocks_per_year / block_per_year
```

where

```
real_blocks_per_year = 31561920 / real_block_time
```
and 

```
real_block_time = timt_for_n_blocks / n
```

so, the final equation for non-epoch networks APR is

```
APR = (annual_provisions * (1 - community_tax) / bonded_tokens_amount) * correction_annual_coefficient
```

for epoch networks is:

```
APR = annual_provisions * (1 - community_tax) / bonded_tokens_amount
```

bro-n-bro-0 reviewed (changes requested):
Added in the comment above

bro-n-bro-0 commented:
Also, mention please share if not all tokens inflate in honor of staking

bro-n-bro-2 committed:
added unify formula for epoch, non -epoch chains. Added correction_annual_coefficient

added unify formula for epoch, non -epoch chains. Added correction_annual_coefficient, minor UI fixes
